routers/web/org/projects.go

```diff
@@ -104,7 +105,7 @@ func Projects(ctx *context.Context) {
 	}
 
 	for _, project := range projects {
-		project.RenderedContent = project.Description
+		project.RenderedContent = templates.Str2html(project.Description) // FIXME: is it right? why not render?
```

Comment:
well it would be nice to know why not ...

Reply:
But it is not in this PR's scope. It is just the old logic.